backend/server/src/db/cmd.rs

```diff
@@ -86,3 +99,18 @@ async fn run_script(db: &Db, script_path: &Path) -> Result<()> {
 
     Ok(())
 }
+
+fn console(config: &config::Db) -> Result<NeverReturns, io::Error> {
+    Err(Command::new("psql")
+        .args(&[format!(
+            "postgresql://{}:{}@{}:{}/{}",
+            config.user,
+            config.password.expose_secret(),
+            config.host,
+            config.port,
+            config.database,
+        )])
+        .exec())
```

Comment:
I would much rather prefer this to be multiple statements. This deep nesting should be avoided if possible. Additionally, a nicer error reporting would be nice as well.

```rust
let url = format!(
    "postgresql://{}:{}@{}:{}/{}",
    config.user,
    config.password.expose_secret(),
    config.host,
    config.port,
    config.database,
);

let e = Command::new("psql").arg(&url).exec();
if e.kind() == io::ErrorKind::NotFound {
    eprintln!("command 'psql' not found: make sure it is installed and in your PATH");
}

Err(e)
```

Alternatively, we could package the extra error info with the error. In that case, return `anyhow::Result` instead of `io::Result` and in the `if` return an error with `.context()`, without context otherwise.

Reply:
I did it slightly differently, still. You might complain about the formatting, again; I don't know. But I think we actually want a nice message for every error kind. I just made it generic in the cases where I don't know why `exec` would fail with them (and it might be that it never happens).

Reply:
Looks good to me!